sanity-io/structured-content-2022
# Add ids to Sanity-sourced headings

# Add ids to Sanity-sourced headings

## Intent

- Delete `<Heading />` usage and component, fix #41 
- Add `id` props to all `h2`/`h3`s output from _Hero.tsx_ (maybe unnecessary?), _RichText.tsx_ and _TextAndImage.tsx_

## Description

I am assuming here that it's not necessary to have _all_ headings have an ID, but only those headings that are added by the editors in Sanity. Meaning that e.g. it doesn't make sense to me to be able to anchor-link to the `<h2 />`s output by `<AccordionSection />`.

## Testing this PR

1. Verify that the approach taken in this PR makes sense and is in accordance to what was discussed in today's "go/no-go" meeting
2. Open the [frontpage](https://structured-content-2022-web-pdjjhj9iy.sanity.build/)
3. Inspect the "Join us in-person or online", "Schedule" and "Sponsor Structured Content 2022" headings and verify that they are rendered with unique `id` attributes
4. Navigate around the various pages and verify that you get no console output warning about duplicate IDs, or any other errors/warnings for that matter
